Modules/AwardDKP.lua

```diff
@@ -0,0 +1,105 @@
+local _, core = ...;
+local _G = _G;
+local MonDKP = core.MonDKP;
+local L = core.L;
+
+local function GetEligibleGuildMembers(onlineOnly, sameZone, currZone)
+    -- Get list of guild members, optionally filtered for only online members and/or members currently in zone currZone
+    -- Returns table playerTable where playerTable[playerName] is not nil iff playerName is eligible
+    local playerTable = {}
+    for playerIndex = 1, GetNumGuildMembers() do
```

Comment:
Make sure `GuildRoster()` is called before this call -> https://wowwiki.fandom.com/wiki/API_GuildRoster

Reply:
```suggestion
    GuildRoster()
    for playerIndex = 1, GetNumGuildMembers() do
```